### DISCUSSION TITLE
Next steps

### DISCUSSION THREAD

**Discussion by mdubinko** [mdubinko/gsoc-contrib#10] [Ideas]
I recently accepted a pull request with more teardown images. (Thanks @loredous !) I'm hoping to get some more details on his progress.

Meanwhile, open discussion thread for what needs to happen next. Ideas great and small welcome.

**Comment by loredous** [comment]
Thanks for the invite! At the moment, I'm trying to approach the GSOC from the perspective of getting a working Buildroot toolchain created. I've managed to be able to replace the rootfs partition on the "official" SD card, which then allows me to run different software utilities. The biggest issue I am running into at the moment is that I have near zero experience with the uBoot side of things, and haven't been able to build a working uBoot kernel image that will boot on the GSOC, so I'm stuck with only the official drivers and kernel. I will update the wiki/repo with my current findings and continue plugging away at getting a full working Buildroot configuration.

**Reply by NeonFux** [reply]
Have you seen https://github.com/d3cker/gsoc-puhumod Debian on GSOC
